spec/cucumber/core/test/filters/locations_filter_spec.rb

```diff
@@ -200,9 +200,9 @@ def test_case_named(name)
             filter = described_class.new([scenario_location, another_scenario_location, whitespace_location])
             compile([doc], receiver, [filter])
             expect(receiver.test_case_locations).to eq([
-              test_case_named('one').location,
-              test_case_named('two').location
-            ])
+                                                         test_case_named('one').location,
```

Comment:
You should be able to have these 1 indented.

Reply:
This should work
```
expect(receiver.test_case_locations).to eq(
  [test_case_named('one').location, test_case_named('two').location]
)
```

Reply:
Honestly, I think that's worse, because its less clear that there are two elements in the array, and we lose symmetry with the multiline input arrays. For example:

```ruby
      it 'sorts by the given locations' do
        locations = [
          Test::Location.new('features/test.feature', 6),
          Test::Location.new('features/test.feature', 3)
        ]
        ...
        expect(receiver.test_case_locations).to eq([
          Test::Location.new('features/test.feature', 3),
          Test::Location.new('features/test.feature', 6)
        ])
      end
```ruby

I gotta say, I'm not super sold on Rubocop's utility, overall. I recognize the value of having an automated single style enforcer in an open source project with many different contributors, but here we are, having a style discussion anyways, despite Rubocop being here, trying to figure out how to game around it.

Reply:
Here's a concrete suggestion, rather than just me complaining :)

How about we disable (or make an exception for) the rule that prompted the excessive indentation in the first place?